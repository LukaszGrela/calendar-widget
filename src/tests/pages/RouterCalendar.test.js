import React from 'react';
import {shallow} from 'enzyme';
import RouterCalendar from '../../pages/RouterCalendar';

test('should render correctly', () => {
    const wrapper = shallow(<RouterCalendar />);

    expect(wrapper).toMatchSnapshot();
});

test('should render correctly with props', ()=> {

    const wrapper = shallow(<RouterCalendar 
        match={
            {
                params: {
                    year:'1979',
                    month:'6',
                    date:'13'
                }
            }
        }
    />);

    expect(wrapper).toMatchSnapshot();
});