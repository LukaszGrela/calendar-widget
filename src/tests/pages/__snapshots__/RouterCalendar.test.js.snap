// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<section>
  <article>
    <p>
      React Calendar Widget example - calendar date is fed from the route
    </p>
  </article>
  <article
    className="widgets"
  >
    <CalendarWidget
      currentMonth={"1970-01-01T00:00:00.000Z"}
      onDateChanged={[Function]}
      todayDate={"1970-01-01T00:00:00.000Z"}
    />
  </article>
  <nav>
    <NavLink
      activeClassName="active"
      ariaCurrent="true"
      to="/"
    >
      Home
    </NavLink>
  </nav>
</section>
`;

exports[`should render correctly with props 1`] = `
<section>
  <article>
    <p>
      React Calendar Widget example - calendar date is fed from the route
    </p>
  </article>
  <article
    className="widgets"
  >
    <CalendarWidget
      currentMonth={"1979-06-12T23:00:00.000Z"}
      onDateChanged={[Function]}
      todayDate={"1970-01-01T00:00:00.000Z"}
    />
  </article>
  <nav>
    <NavLink
      activeClassName="active"
      ariaCurrent="true"
      to="/"
    >
      Home
    </NavLink>
  </nav>
</section>
`;
